README.md

```diff
@@ -127,24 +130,25 @@ docker-compose -f docker-compose_dev.yml up -d  # Run the OTE Services (detached
 Note that default values will be used if certain environment variables are not present.
 To inspect which environment variables can be specified, please inspect the [Docker Compose file](docker-compose_dev.yml).
 
-This Docker Compose file will use your local files for the application, meaning updates in your local files (under `app/`) should be reflected in the running application after hypercorn reloads.
+This Docker Compose file will use your local files for the application, meaning updates in your local files (under `app/`) should be reflected in the running application upon storing the changes to disk, after hypercorn reloads.
 You can go one step further and use your local files also for the `oteapi-core` repository by specifying the `PATH_TO_OTEAPI_CORE` environment variable:
 
```

Comment:
It would probably be even more useful if one could specify path to local versions of plugins one want to include.

Reply:
Hmmm... yes that would also be very beneficial!
Let's make a separate issue for it perhaps, as this is a developer thing, while this PR implements something general that should also be used in production.

Reply:
Created issue #76